src/StepDefinition/MinkElementTrait.php

```diff
@@ -129,4 +129,18 @@ public function pressButtonBelowElement($button, $locator, $selector = 'css') {
     $element->pressButton($button);
   }
 
+  /**
+   * @Then Value of input field :locator is :value
```

Comment:
should have a second @Then line which makes it possible to change $selector to xpath, see other steps in the same file for example.